src/routes/data/get.rs

```diff
@@ -197,6 +215,7 @@ pub fn with_token<S: SessionStore, R: Renderer, T: TokenGenerator, H: Storer>(
                             css: query_params.css,
                             edit: query_params.edit,
                             relay_url: query_params.relay_url,
+                            js_message: query_params.js_message,
```

Comment:
Sanitization should happen on this route as well, even if it's been done in the without_tokens route and should be sanitized already